scripts/plot_performance_over_time.py

```diff
@@ -88,25 +91,34 @@ def lighten_color(input_color, amount=0.5):
         single_run_color = [colors.to_rgba(lighten_color(color, 0.7), 0.5)]
 
         # Filter the runs that belong to this benchmark item
-        filtered_df = run_durations_df[run_durations_df['name'] == benchmark_name]
+        filtered_df = run_durations_df[run_durations_df["name"] == benchmark_name]
 
         # Plot runs into combined graph
         for success in [True, False]:
-            data = filtered_df[filtered_df['success'] == success]
+            data = filtered_df[filtered_df["success"] == success]
             if not data.empty:
-                data.plot(ax=ax, kind='scatter', x='begin', y='duration', c=single_run_color,
-                          figsize=(figw, figh), s=5, marker=('o' if success else 'x'), linewidth=1)
+                data.plot(
+                    ax=ax,
+                    kind="scatter",
+                    x="begin",
+                    y="duration",
+                    c=single_run_color,
+                    figsize=(figw, figh),
+                    s=5,
```

Comment:
replace with long form? I can't remember what s stands for

Reply:
Apparently, it's the marker size. I couldn't find a proper long version for scatter plots and added a comment.